vendors/circom/circomlib/generated/README.md

```diff
@@ -0,0 +1,15 @@
+# Generated
+
+This is taken and modified from [iden3/circom/code_producers/src/c_elements](https://github.com/iden3/circom/tree/v2.1.8/code_producers/src/c_elements).
+
+## Modifications
+
+- Modified to remove errors from compilation.
+- `loadCircuit` and `writeBinWitness` are moved to [common/calcwit.hpp](/vendors/circom//circomlib/generated/common/calcwit.hpp) and [common/calcwit.hpp](/vendors/circom//circomlib/generated/common/calcwit.cpp).
+- `loadWitness` is created instead of `loadJson`, which loads the witness from the `absl::flat_hash_map<>`.
+
+See the followings for more details.
```

Comment:
``` suggestion
See the following files for more details on the modifications.
```
["Following" is an adjective](https://www.quora.com/What-should-I-say-Which-of-the-following-or-Which-of-the-followings) and, therefore, never plural,